setup.py

```diff
@@ -98,6 +98,18 @@
     DPCPP_LIBS = []
 DAAL_DEFAULT_TYPE = 'double'
 
+
+#Level Zero workaround for oneDAL Beta06
+from generator.parse import parse_version
+
+header_path = daal_root + '/include/services/library_version_info.h'
+
+
+with open(header_path) as header:
+    v = parse_version(header)
+    dal_build_version = (int(v[0]), int(v[2]))
+    print('Found DAAL version {}.{}'.format(*dal_build_version))
```

Comment:
We already have such print here
https://github.com/IntelPython/daal4py/blob/master/generator/gen_daal4py.py#L183
In that place it shows that all headers are read successfully.
Two same prints in the log can make misunderstanding.
Maybe we should change the message or remove it at all.

Reply:
removed